docs/Rollup.md

```diff
@@ -347,7 +347,7 @@ Defaults to `@npm//@bazel/rollup/bin:rollup-worker`
 
 (*Boolean*): Whether to execute the rollup binary with the --silent flag, defaults to False.
 
-Using --silent can cause rollup to [ignore errors/warnings](https://github.com/rollup/rollup/blob/master/docs/999-big-list-of-options.md#onwarn) 
```

Comment:
Lets revert this...

Reply:
Sorry, can you elaborate why? The trailing whitespace here is a noop and inconsistent with the rest of the file.

Reply:
Personally I would disagree with this. This is a good drive-by fix, similar to fixing typos while spotting them. It's unlikely these things would ever be addressed if people should send a dedicated PR. A separate commit is another story, but quite a nitpick.

Reply:
If you were changing the file already then sure, but you're not evening touching the file 🤷

Reply:
This file is auto-generated from the `rollup_bundle.bzl` file and had to be updated..

Reply:
Well nm then 👍 

Interesting it got checked in like this at all then 🤔